src/KM_Controls.pas

```diff
@@ -46,10 +46,9 @@   TKMMasterControl = class
     
     function GetNextCtrlID: Integer;
   public
-    constructor Create;
     destructor Destroy; override;
 
-    property MainPanel: TKMPanel read fCtrl;
+    property MasterPanel: TKMPanel read fMasterPanel;

```

Comment:
1. Renamed TKMMasterControl.fCtrl into TKMMasterControl.fMasterPanel as it means exactly master panel (main panel)